clientv3/client.go

```diff
@@ -215,6 +216,11 @@ func (c *Client) dialSetupOpts(endpoint string, dopts ...grpc.DialOption) (opts
 	if c.cfg.DialTimeout > 0 {
 		opts = []grpc.DialOption{grpc.WithTimeout(c.cfg.DialTimeout)}
 	}
+	if c.cfg.DialKeepAlive > 0 {
+		opts = append(opts, grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			Time: c.cfg.DialKeepAlive * time.Second,
```

Comment:
`DialTimeout` and `AutoSyncInterval` are also of type `time.Duration` but they do not multiply by `time.Second`-- this is inconsistent; just dropping the multiply should be fine